.circleci/config.yml

```diff
@@ -142,18 +137,29 @@ jobs:
           name: Create checksums
           working_directory: artifacts
           command: sha256sum * > checksums.txt && cat checksums.txt
-      - run:
-          name: Publish artifacts on GitHub
-          command: |
-            TAG="$CIRCLE_TAG"
-            TITLE="$TAG"
-            BODY="Build artifacts generated at this tag."
-            ghr -t "$GITHUB_TOKEN" \
-              -u "$CIRCLE_PROJECT_USERNAME" -r "$CIRCLE_PROJECT_REPONAME" \
-              -c "$CIRCLE_SHA1" \
-              -n "$TITLE" -b "$BODY" \
-              -delete \
-              "$TAG" ./artifacts/
+      - store_artifacts:
+          path: artifacts
+      # Deploy to GitHub releases on tag builds
+      - when:
+          condition: << pipeline.git.tag >>
```

Comment:
Nice. Does the empty string amounts to `false` here?

Reply:
Exactly